share/spice/crunchbase/collections/crunchbase_collections.js

```diff
@@ -0,0 +1,66 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_crunchbase_collections = function(api_result){
+        // throw error if no result, error result, no items, or array of 0 items
+        if (!api_result || api_result.error || !DDG.getProperty(api_result, 'data.items') || api_result.data.items.length === 0) {
+            return Spice.failed('crunchbase');
+        }
+        var data = api_result.data, // get org data
+	    items = data.items, // get all orgs
+	    firstResult = items[0], // get first org to render for now
+	    permalink = firstResult.path; // get endpoint of first org
+
+	var metadata = api_result.metadata; // get metadata for base urls
+	// get base urls if metdata exists
+	if (metadata){
+            var wwwPathPrefix = metadata.www_path_prefix,
+                imagePathPrefix = metadata.image_path_prefix;
+	}
+
+        // get organization information from organization endpoint
+	$.getJSON('js/spice/crunchbase/organizations/'+ permalink.split('/')[1],function(response) {
```

Comment:
@moollaza Looking at the [docs](https://developer.crunchbase.com/docs) for the organizations endpoint again, these are the the paramaters it makes sense to search by. We're currently using 'query', which is why I started out using the tile view until you advised not to over email. It's imprecise. (Also, I took a screenshot of all available orgs parameters and put it at the bottom of this comment for reference):

![screenshot 2015-07-06 14 35 26](https://cloud.githubusercontent.com/assets/1807895/8529993/4abf68d2-23ec-11e5-81e3-07d52dc35478.png)
We could do something like:
'Crunchbase name $COMPANYNAME' -> to hit the name option, info view
'Crunchbase $ORGANIZATIONTEXT' -> hit the query option, tile view

and then there's a category parameter as well that requires us to get the available categories and their UUIDs from the categories endpoint beforehand:
![screenshot 2015-07-06 14 37 31](https://cloud.githubusercontent.com/assets/1807895/8530046/a2997278-23ec-11e5-8dbe-b42aff4bd4cd.png)



Right now the paging data for the categories endpoint is:
```
  "paging": {
   "total_items": 864,
   "number_of_pages": 1,
   "current_page": 1,
   "sort_order": "name ASC",
   "items_per_page": 5000,
   "revision": 1436207068
  }
```

I could add categories, but it would take substantially more time than using the query parameter.

And to save time, here's all the parameters for the organizations endpoint for reference as a screenshot:
![screenshot 2015-07-06 14 44 51](https://cloud.githubusercontent.com/assets/1807895/8530204/9af7a1b0-23ed-11e5-91b0-e1c9474a5439.png)